taxonomies/ingredients.txt

```diff
@@ -54384,6 +54440,10 @@ sv:vaniljstång
 carbon_footprint_fr_foodges_ingredient:fr:Gousse de Vanille
 carbon_footprint_fr_foodges_value:fr:2.6
 
+<en:vanilla
+en:vanilla vein
+es: vaina de vainilla, vainas de vainilla
```

Comment:
Those should go as synonyms of this entry:

<en:vanilla
en:vanilla pod, vanilla pods
de:Vanilleschoten
es:vainas de vainilla

Reply:
ok